Week1/node.js

```diff
@@ -0,0 +1,184 @@
+const mysql = require('mysql');
+const database = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+database.connect();
+
+// create database world
+
+database.query(`CREATE DATABASE IF NOT EXISTS world`, (error, result) => {
+  if (error) throw error;
+  console.log('world database is created');
+});
+
+// use world database
+database.query(`USE world`, (error, result) => {
+  if (error) throw error;
+  console.log('change to WORLD database');
+});
+
+//========== create tables : country , city , and insert data in it =================
+
+const table_of_country =
+  'CREATE TABLE IF NOT EXISTS country (name VARCHAR(50), continent VARCHAR(50), region VARCHAR(50), surface_area FLOAT, indep_year INT, population INT, life_expectancy, GNP FLOAT, GNPold FLOAT, local_name VARCHAR(50), government_form VARCHAR(50), head_of_state VARCHAR(50),capital VARCHAR(50))';
+
+database.query(table_of_country, (error, result) => {
+  if (error) throw error;
+  // console.log(result);
+  console.log('Country table is created');
+});
+
+var table_of_city =
+  'CREATE TABLE IF NOT EXISTS city (id INT, name VARCHAR(50),country_code VARCHAR(10), district VARCHAR(50), population INT)';
+
+database.query(table_of_city, (error, results) => {
+  if (error) throw error;
+  console.log('City table is created');
+});
+
+var insert_countries = [
+  "INSERT INTO country (name, continent,surface_area, population) VALUES ('Angola','Africa',1246700,12878000)",
```

Comment:
did you know that you can have multiple values in a single query
``` 
"INSERT INTO country (name,     continent,  surface_area,  population) VALUES "+
"                    ('Angola', 'Africa',   1246700,       12878000),   "+
"                    ('Syria',  'Asia',     185180,        18269868),   "+ // and so forth
```

Reply:
indeed i did not know , but it is implemented now , thanks a lot.

Reply:
I did not know,  i implement it